db/app.js

```diff
@@ -0,0 +1,19 @@
+
+const express = require('express')
+const { getTopicsController } = require('./controller/controller.js')
+const { handle404 } = require('./errorHandlers/errorHandlers.js')
+
+
+const app = express()
+
+app.use(express.json())
```

Comment:
We don't need this line yet as we're not doing a post request where we need access to the request body
It's best to only include code in a ticket/task that meets the needs of that ticket/task, so could you please remove this from the PR

Reply:
Is this for line 9, in that case I'll remove it